sphinx_gallery/scrapers.py

```diff
@@ -214,7 +239,8 @@ def save_figures(block, block_vars, gallery_conf):
                             % (scraper, type(rst), rst))
         n_new = len(image_path_iterator) - prev_count
         for ii in range(n_new):
-            current_path = image_path_iterator.paths[prev_count + ii]
+            current_path, _ = _find_image_ext(
```

Comment:
so image path iterator outputs `.png`, but find_image_paths just doesn't use the extension when searching for images?

Reply:
the iterator always gives `.png` filenames yes but SG will search for outputs ending in `.png` or `.svg` currently